app/src/androidTest/java/ch/sdp/vibester/activity/AuthenticationActivityTest.kt

```diff
@@ -127,9 +141,9 @@ class AuthenticationActivityTest {
         onView(withId(R.id.username)).perform(ViewActions.typeText(username), closeSoftKeyboard())
         onView(withId(R.id.password)).perform(ViewActions.typeText(password), closeSoftKeyboard())
         onView(withId(R.id.createAcc)).perform(click())
-        Thread.sleep(sleepTime)
-        onView(withId(R.id.email)).check(matches(withText(randomInt.toString().plus("@gg.com"))))
+        Thread.sleep(1000)
```

Comment:
Maybe change to 1000 the sleepTime constant you defined and use that when calling Thread.sleep.

Reply:
will do!